addons/website_payment/static/tests/tours/donation.js

```diff
@@ -68,6 +74,91 @@ registerWebsitePreviewTour('donation_snippet_edition', {
             trigger: ":iframe input#other_amount:not(:checked)",
             run: () => {}, // This is a check
         },
+        {
+            content: "Click on the custom amount radio button",
+            trigger: ":iframe input#other_amount",
+            run: "click"
+        },
+        {
+            content: "Verify the display of a warning message",
+            trigger: ":iframe p#warningMessageId",
+        },
+        {
+            content: "Enter an amount less than the minimum value",
+            trigger: ":iframe input#other_amount_value",
+            run: "edit 1",
+        },
+        {
+            content: "Verify the display of a minimum value warning message",
+            trigger: ":iframe p#warningMinMessageId",
+        },
+        {
+            content: "Click on the first radio button",
+            trigger: ":iframe input[name='o_donation_amount']:first-child",
+            run: "click"
+        },
+        {
+            content: "Ensure the custom amount value is cleared",
+            trigger: ":iframe input[name='o_donation_amount']:first-child",
+            run: () => {
+                const iframe = document.querySelector("iframe").contentDocument;
+                const warningEl = iframe.querySelector("#warningMinMessageId");
+                const customAmountEl = iframe.querySelector("#other_amount_value");
+                if (!warningEl.classList.contains("d-none") || customAmountEl.value != "") {
+                    console.error("Warning message should not be display");
+                }
+            }
```

Comment:
Here if your objective is to check whether the custom amount is cleared, dont you think your error should also be making sure the "amount should be cleared" ?

Reply:
Changed !!